js/sign/src/cli.ts

```diff
@@ -0,0 +1,49 @@
+import commander from 'commander';
+import { IntegrityBlockSigner } from './integrityblock.js';
+import * as fs from 'fs';
+import crypto, { KeyObject } from 'crypto';
+
+function readOptions() {
+  return commander
+    .requiredOption(
+      '-i, --input <file>',
+      'input unsigned web bundle (required)'
+    )
+    .requiredOption(
+      '-k, --privateKey <file>',
+      'path to ed25519 private key (required)'
+    )
+    .option(
+      '-o, --output <file>',
+      'signed web bundle output file',
+      'signed.wbn'
+    )
+    .parse(process.argv);
+}
+
+function readKeyFile(keyPath: string): KeyObject {
+  const file = fs.readFileSync(keyPath);
+  const key = crypto.createPrivateKey({
+    key: file,
+  });
+  return key;
+}
+
+export function main() {
+  const options = readOptions();
+  if (!options.privateKey || !options.input) {
```

Comment:
This is already checked by `commander`.

Reply:
Yes in the case of the CLI but not in the case of the plugins.